blockchain/blockchain.go

```diff
@@ -109,6 +110,17 @@ func (b *Blockchain) StateCommitment() (*felt.Felt, error) {
 	})
 }
 
+// func (b *Blockchain) JunoGetNodesFromRoot(key *felt.Felt) (core.StateReader /*, StateCloser*/, error) {
+// 	b.listener.OnRead("JunoGetNodesFromRoot")
+// 	var state core.StateReader
+// 	// var closer StateCloser
+// 	return state, b.database.View(func(txn db.Transaction) error {
+// 		var err error
+// 		state, _, err = core.NewState(txn).NodeFromRoot(key)
+// 		return err
+// 	})
+// }
+
```

Comment:
Same with this

Reply:
You mentioned about writing some codes in `blockchain.go` file right? (in the message thread)
I was not sure about whether I'm doing correct thing here. So I just commented it.